PresentData/PresentMonTraceConsumer.cpp

```diff
@@ -1299,46 +1292,40 @@ void PMTraceConsumer::RemoveLostPresent(std::shared_ptr<PresentEvent> p)
     RemovePresentFromTemporaryTrackingCollections(p);
 
     // mPresentsByProcess
-    {
-        auto& presentsByThisProcess = mPresentsByProcess[p->ProcessId];
-        auto elementsRemoved = presentsByThisProcess.erase(p->QpcTime);
-        assert(elementsRemoved <= 1);
-    }
+    auto& presentsByThisProcess = mPresentsByProcess[p->ProcessId];
+    auto elementsRemoved = presentsByThisProcess.erase(p->QpcTime);

```

Comment:
Looks like you lost the assert here?

Reply:
Oh, I see your comment now that this was intentional. In that case, I'd remove the `elementsRemoved` variable.